src/core/public/chrome/ui/header/collapsible_nav_group_enabled_top.tsx

```diff
@@ -155,7 +104,9 @@ export const CollapsibleNavTop = ({
           <>
             <EuiSpacer />
             {currentNavGroup?.type ? (
-              <EuiText>{currentNavGroup?.title}</EuiText>
+              <EuiText size="s">
+                <h3>{currentNavGroup?.title}</h3>
```

Comment:
`currentNavGroup?.type` was true so `currentNavGroup` exists. `currentNavGroup?.title` doesn't need the `?`.

Reply:
Thanks, updated.